cmd/commands.go

```diff
@@ -305,6 +305,54 @@ the KEY=VALUE format will be loaded into the Caddy process.
 		},
 	})
 
+	RegisterCommand(Command{
+		Name:  "storage",
+		Short: "Commands for working with Caddy's storage",
+		Long: `
+Allows exporting and importing Caddy's storage contents. The two commands can be
+combined in a pipeline to transfer directly from one storage to another:
+
+$ caddy storage export --config Caddyfile.old --output - |
+> caddy storage import --config Caddyfile.new --input -
+
+With - referring to stdout/stdin.
```

Comment:
```suggestion
The - argument refers to stdout and stdin, respectively.
```

Reply:
I'd put single quotes around it like `'-'` maybe

Reply:
`caddy fmt` does not use quotes so I left them out